### PR TITLE
(RE-12520) 0.99.36 release preparation.

### PR DESCRIPTION
**Pull request by e-gris**

### CODE REVIEW

**Comment by sarameisburger** [comment]
You'll wanna update https://github.com/puppetlabs/packaging/pull/962/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR428 as well

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.99.36] - 2019-07-09
+### Updated
```
Comment (mwaggett) [inline CHANGELOG.md]: (nitpick, also not specific to this PR, BUT) do we want to keep `Updated` as a heading? Looks like the 0.99.34 release is the first time we used it. In the past, we've just used 'Added', 'Changed', 'Fixed', and 'Removed'. ¯\_(ツ)_/¯

**Comment by e-gris** [comment]
D'oh!  I miss that _every_ damn time.

File: CHANGELOG.md (hunk 1 above)
Comment (e-gris) [reply CHANGELOG.md]: Let's fix that now. Don't need 'Updated' with 'Changed' already established.

**Pull request merged**